reactor-core/src/test/kotlin/reactor/core/publisher/FluxExtensionsTests.kt

```diff
@@ -162,7 +162,7 @@ class FluxExtensionsTests {
         StepVerifier
                 .create(IOException()
                         .toFlux<String>()
-                        .onErrorResume(IOException::class, { "foo".toMono() }))
+                        .onErrorResume(IOException::class) { "foo".toMono() })
```

Comment:
could you please improve the readability of this one as well? (as you did [here](https://github.com/reactor/reactor-core/pull/1475/commits/3f29a64a72ed969bad9a703aff8ddc33a33e60af))

Reply:
(I did the update)